packages/behandling-pleiepenger/src/components/MedisinskVilkår.tsx

```diff
@@ -1,7 +1,19 @@
 import * as React from 'react';
 import { MicroFrontend } from '@fpsak-frontend/utils';
+import { useRestApiErrorDispatcher } from '../../../rest-api-hooks';
```

Comment:
Er det ikke mulig å bruke pakkenavn isteden for absolutt import når dere bygger mikrofrontend?

Reply:
Jo, det bør være mulig. Fikser.